poker-pals/src/components/Poker.vue

```diff
@@ -215,10 +205,7 @@ export default {
                 action: action,
                 raiseToValue: this.raiseToValue, // only used if player is raising
             });
-            // TODO: later use v-bind and not getElementById
             this.checkFold = false;
-            let b = document.getElementById("buttonCheckFold");
-            b.style.backgroundColor = '#ffffff'; // color theme for button not selected
```

Comment:
Could b have a more descriptive name?

Reply:
that code was removed

Reply:
Weird.  Totally saw it as an addition.  My mistake.